src/OpenRiaServices.Client.DomainClients.Http/Framework/Http/BinaryHttpDomainClient.cs

```diff
@@ -492,7 +491,7 @@ internal DataContractSerializerSettings GetSubmitDataContractSettings()
                     var method = entityType.GetMethod(entityAction.Name);
                     foreach (var parameter in method.GetParameters())
                     {
-                        var type = parameter.ParameterType;
+                        var type = TypeUtility.GetNonNullableType(parameter.ParameterType);
```

Comment:
Is all nullable?

Reply:
no, but if it is not nullable GetNonNullableType it will return the type passed as the parameter